DBMS/DBMS4/assignment-2.sql

```diff
@@ -0,0 +1,45 @@
+-- View  used to display member name and
+-- all issue details of the member using a simple SELECT command.
+CREATE VIEW BookIssueInfo
+AS
+SELECT m.member_id, m.member_name, bi.issue_date,bi.due_date,bi.accession_no
+FROM members m,book_issue bi
+WHERE m.member_id = bi.member_id ;
+SELECT * FROM BookIssueInfo;
+
+
+
+
+-- View which contains three columns, member name,
+-- member id and full description of category, i.e., Faculty, Students
+-- and Others instead of F,S and O.
+CREATE VIEW MemberInfo
+AS
+SELECT member_id,member_name,IF(category='F','Faculty',IF(category='S','Student','Others'))AS category
+FROM members ;
+SELECT * FROM MembertInfo;
+
+
+
+
+-- View which contains the information – subject name,
+-- title, member name, category, issue date, due date and return
+-- date. If the books have not been returned, NULL should be
+-- displayed instead of return date
+-- to take all rows from book return
+CREATE VIEW CompleteInformation
+AS
+SELECT s.subject_name,t.title_name ,m.member_name,m.category,bi.issue_date,bi.due_date,br.return_date  
+FROM titles t,subjects s,members m,books b,book_issue bi
```

Comment:
Can use natural join so that less no of cases have to be checked which you have done